doc/classes/NavigationServer2D.xml

```diff
@@ -133,6 +133,14 @@
 				Create a new map.
 			</description>
 		</method>
+		<method name="map_force_update">
+			<return type="void" />
+			<argument index="0" name="map" type="RID" />
+			<description>
+				[b]Note:[/b] With power comes responsibility. This function should only be used by users that really know what they are doing and have a good reason for it. Forcing an immediate update of a navigation map requires locking the NavigationServer and flushing the entire NavigationServer command queue. Not only can this severely impact the performance of a game but it can also introduce bugs if use inappropriately without much foresight.
+				This function immediately forces synchronisation of the specified navigation [code]map[/code] [RID]. By default navigation maps are only synchronized at the end of each physics frame. This function can be used to immediately (re)calculate all the navigation meshes and region connections of the navigation map. This makes it possible to query a navigation path for a changed map immediately and in the same frame (multiple times if needed). Due to technical restrictions the current NavigationServer command queue will be flushed. This means all already queued update commands for this physics frame will be executed, even those intended for other maps, regions and agents not part of the specified map. The expensive computation of the navigation meshes and region connections of a map will only be done for the specified map. Other maps will receive the normal synchronisation at the end of the physics frame. Should the specified map receive changes after the forced update it will update again as well when the other maps receive their update. Avoidance processing and dispatch of the [code]save_velocity[/code] signals is untouched by this function and continues to happen for all maps and agents at the end of the physics frame.
```

Comment:
Changing `synchronisation` to en_US spelling, and typo in `save_velocity`.

I'd also suggest splitting the huge paragraph in multiple ones, if anything so that it's more digest in the docs and also in the XML. Not sure if the line breaks I suggest are the best, the idea generally is that one main concept == one paragraph.

```suggestion
				This function immediately forces synchronization of the specified navigation [code]map[/code] [RID]. By default navigation maps are only synchronized at the end of each physics frame. This function can be used to immediately (re)calculate all the navigation meshes and region connections of the navigation map. This makes it possible to query a navigation path for a changed map immediately and in the same frame (multiple times if needed).
				Due to technical restrictions the current NavigationServer command queue will be flushed. This means all already queued update commands for this physics frame will be executed, even those intended for other maps, regions and agents not part of the specified map. The expensive computation of the navigation meshes and region connections of a map will only be done for the specified map. Other maps will receive the normal synchronization at the end of the physics frame. Should the specified map receive changes after the forced update it will update again as well when the other maps receive their update.
				Avoidance processing and dispatch of the [code]safe_velocity[/code] signals is untouched by this function and continues to happen for all maps and agents at the end of the physics frame.
```